ZUGFeRD/InvoiceDescriptor21Writer.cs

```diff
@@ -738,10 +738,13 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream)
                         Writer.WriteElementString("ram:ProprietaryID", account.ID);
                     }
                     Writer.WriteEndElement(); // !PayeePartyCreditorFinancialAccount
-
-                    Writer.WriteStartElement("ram:PayeeSpecifiedCreditorFinancialInstitution");
-                    Writer.WriteElementString("ram:BICID", account.BIC);
-                    Writer.WriteEndElement(); // !PayeeSpecifiedCreditorFinancialInstitution
+                    
+                    if (!String.IsNullOrEmpty(account.BIC))
+                    {
+                        Writer.WriteStartElement("ram:PayeeSpecifiedCreditorFinancialInstitution");
```

Comment:
could you add a unit test for this? Imho, it would be sufficient to check if the raw xml output contains 'ram:BICID' if it not specified.